public/index.html

```diff
@@ -4,6 +4,8 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:400,500,600,700,800">
+    <link rel="stylesheet" href="http://www.material-ui.com/css/font-icons/style.css">
```

Comment:
Since this is served over `http` and we're using `https` in Netlify the request is being blocked by the browser (cannot load http resources if the site loads in https). I've checked the link and they have an SSL certificate issued for GitHub but they serve it in the material-ui domain, so that's causing problems too. Maybe we should just download the stylesheet in the project?